docs/clone.md

```diff
@@ -0,0 +1,30 @@
+# Clone
+
+## Using `clone`
+
+```javascript
+import clone from 'consus-core/clone';
+
+let original = {
+    a: [0, 1, 2],
+    b: 'Hello, World!',
+    c: 3.14159265
+};
+
+let twin = clone(original);
+
+twin;
+/*
+{
+    a: [0, 1, 2],
+    b: 'Hello, World!',
+    c: 3.14159265
+}
+*/
+
+twin === original;
+// false
+
+twin.a === original.a;
+// false
```

Comment:
Buy twin.b === original.b will be true and twin.c === original is true. Can we add that to the doc?